javascript/javascript1/week1/Ez_Namey.js

```diff
@@ -0,0 +1,5 @@
+const firstWords = [ "Easy" , "Awesome" , "Corporate" , "Highly" , "Brownish", "Underneath" , "Wonderful" , "Fabulous" , "Highlighted", "Commited"];
+const secondWords = [ "Golden" , "Silver" , "Ultimate" , "Absolute" , "Sky" , "Earth" , "Planet", "Panda", "Bear" , "Wolves"];
+let startupName= "";
+startupName = firstWords[Math.floor(Math.random() * 10) + 0] +" "+secondWords[Math.floor(Math.random() * 10) + 0];
+console.log("The startup: "+`"${startupName}"`+" contains " +startupName.length+" characters");

```

Comment:
You don't need to mix string interpolation and string concatenation, better to pick one or the other, like this:
```js
console.log(`The startup: "${startupName}" contains ${startupName.length} characters`);
```